api/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -580,18 +580,19 @@ type FLPMetrics struct {
 	// +optional
 	IncludeList *[]FLPMetric `json:"includeList,omitempty"`
 
-	// `disableAlerts` is a list of alert groups that should be disabled from the default set of alerts.
+	// `disableHealthRules` is a list of health rule templates that should be disabled from the default set.
 	// Possible values are: `NetObservNoFlows`, `NetObservLokiError`, `PacketDropsByKernel`, `PacketDropsByDevice`, `IPsecErrors`, `NetpolDenied`,
 	// `LatencyHighTrend`, `DNSErrors`, `ExternalEgressHighTrend`, `ExternalIngressHighTrend`, `CrossAZ`.
-	// More information on alerts: https://github.com/netobserv/network-observability-operator/blob/main/docs/Alerts.md
+	// More information on health rules: https://github.com/netobserv/network-observability-operator/blob/main/docs/Alerts.md
 	// +optional
-	DisableAlerts []AlertTemplate `json:"disableAlerts"`
+	DisableHealthRules []HealthRuleTemplate `json:"disableHealthRules"`
```

Comment:
I think for disabling, we should keep the existing API and continue to only affect alerts. Disabling is mostly to help users when they have too much alerting noise, and it's not so needed for recording rules
Also because we should not rename this field because it already existed before the TP feature, that would be a breaking change.